arduino_vault.ino

```diff
@@ -2,48 +2,46 @@
-WebSocketsServer webSocket = WebSocketsServer(12346);    // the websocket uses port 81 (standard port for websockets
+WebServer server(80);                                 // the server uses port 80 (standard port for websites
+WebSocketsServer webSocket = WebSocketsServer(81);    // the websocket uses port 81 (standard port for websockets
 int interval = 1000;                                  // send data to the client every 1000ms -> 1s
 unsigned long previousMillis = 0;                     // we use the "millis()" command for time reference and this will output an unsigned long
-
-
-String webpage ="<!DOCTYPE html><html><head><title>Page Title</title></head><body style='background-color: #eeeeee'><span style='color: #003366'><h1>Lets generate a random number</h1><p>The first random number is: <span id='rand1'>-</span></p><p>The second random number is: <span id='rand2'>-</span></p><p><h1>Zdarov gadi</h1><form><input type='text' id='laukas'><br><br></form><button type='button' id='BTN_SEND_BACK'>Send info to ESP32</button></p></span></body><script>;var Socket;document.getElementById('BTN_SEND_BACK').addEventListener('click',button_send_back);function init(){Socket=new WebSocket('ws://'+window.location.hostname+':81/');Socket.onmessage=function(n){processCommand(n)}};function button_send_back(){let laukas=document.getElementById('laukas').value;var n={laukas};Socket.send(JSON.stringify(n))};function processCommand(e){var n=JSON.parse(e.data);document.getElementById('rand1').innerHTML=n.rand1;document.getElementById('rand2').innerHTML=n.rand2;console.log(n.rand1);console.log(n.rand2)};window.onload=function(n){init()};</script></html>";
-
-void printData(){
-//    Serial.println("Date time:"+printLocalTime());
-  //  Serial.println("Sensor value:"+rvalue);
-    Serial.print("HTTP Response code: ");
-   // Serial.println(httpResponseCode);
+/*
+void printData() {
+  Serial.println("Date time:"+printLocalTime());
+  Serial.println("Sensor value:"+rvalue);
+  Serial.print("HTTP Response code: ");
+  Serial.println(httpResponseCode);
 }
-void sendHttpJSON(String value, String date,String sensorId){
-    WiFiClient client;
-    HTTPClient http;
-    http.begin("https://messuring-vault.herokuapp.com/api/");
-    http.addHeader("Content-Type", "application/json");
-    String json = "{\"value\":\""+
-        value  +"\",\"sensor\":\"1\",\"date\":\""+
-    date+"\"}";
-    int httpResponseCode = http.POST(json);
-    Serial.println(json);
-    Serial.println(http.POST(json));
-    Serial.println(httpResponseCode);
-    http.end();
-  
+*/
+void sendHttpJSON(String value, String date, String sensorId) {
+  WiFiClient client;
+  HTTPClient http;
+  http.begin("https://messuring-vault.herokuapp.com/api/");
+  http.addHeader("Content-Type", "application/json");
+  String json = "{\"value\":\"" +
+                value  + "\",\"sensor\":\"1\",\"date\":\"" +
+                date + "\"}";
```

Comment:
There should be string builder or something to build strings nicely

Reply:
For some reason JSON deseralizer doesn't work, HTTP Post doesn't work with string build from deserialized JSON,  since the functionality is there  will leave this issue for later and move on to other stuff